torch/testing/_creation.py

```diff
@@ -7,6 +7,7 @@
 import math
 import collections.abc
 
+complex_float_map = {torch.complex32: torch.half, torch.complex64: torch.float, torch.complex128: torch.double}
```

Comment:
Prefix private datastructures with an underscore and add a comment explaining what this map is for